com.unity.uiwidgets/Editor/UIWidgetsEditorPanel.cs

```diff
@@ -108,6 +108,13 @@ void OnGUI()
                 var pos = _getPointerPosition(evt.mousePosition);
                 _wrapper.OnMouseMove(pos);
             }
+            else if (evt.type == EventType.ScrollWheel && evt.delta.magnitude != 0) {
+                var delta = evt.delta;
+                delta.x  = 0f;
```

Comment:
why making it 0? we need accept and process non-zero delta x too

Reply:
ok, I'll fix it.